addons/sourcemod/scripting/include/movement.inc

```diff
@@ -111,7 +111,8 @@ stock void Movement_GetOriginEx(int client, float result[3])
 	if (TR_DidHit(trace))
 	{
 		TR_GetEndPosition(result, trace);
-		result[2] = result[2] - 0.03125; // Get rid of the offset (CS:GO quirk?)
```

Comment:
This makes it so it's accurate with Source's traces. 
This trace moves the result down 0.03125 but I don't think it should because that was the correct origin of the player.

While it's true that the player is usually 0.03125 above the ground, there are things you can do that lets you be less than 0.03125 off the ground (eg. using a ladder or swimming in water). 

This happens where the trace destination is closer than 0.03125 unit from the ground, but doesn't touch it. In that case, GetOriginEx might return an origin that is inside the ground.

GOKZ jumpstats should already be modified to prevent any potential issue caused by this PR.